debian/additions/mariadb.conf.d/50-client.cnf

```diff
@@ -0,0 +1,28 @@
+#
+# This group is read by the client library
+# Use it for options that affect all clients, but not the server
+#
+
+[client]
+# Default is Latin1, if you need UTF-8 set this (also in server section)
+default-character-set = utf8mb4
+
+# default port
+port = 3306
+
+# socket location
+socket = /var/run/mysqld/mysqld.sock
```

Comment:
note server config path for `socket` doesn't begin with `/var` I'm sure they probably symlink but consistent might be better. `INSTALL_UNIX_ADDRDIR_DEB` in cmake/install_layout.cmake includes `/var` so maybe remove the  '/var' prefix there too. This default unix socket layout propagates to libmariadb to the default path in the client libraries and client packages that build against it.

Reply:
This is how it used to be in my.cnf as well, but you are right, current form should be /run/.. not /var/run/ so I fixed those in the second commit.